Source/Android/jni/SkylanderConfig.cpp

```diff
@@ -124,7 +125,10 @@ Java_org_dolphinemu_dolphinemu_features_skylanders_SkylanderConfig_createSkyland
   std::string file_name = GetJString(env, fileName);
 
   auto& system = Core::System::GetInstance();
-  system.GetSkylanderPortal().CreateSkylander(file_name, sky_id, sky_var);
+  IOS::HLE::USB::SkylanderFigure* figure = new IOS::HLE::USB::SkylanderFigure(file_name);
```

Comment:
This seems a bit odd that we create the figure only to delete it.  Is it to create and save the file?

Reply:
Yes it is

Reply:
I'm also not really able to work on android. I'm using @deReeperJosh's android fixes for this pr